contrib/localnet/orchestrator/smoketest/main.go

```diff
@@ -110,8 +110,15 @@ var LocalCmd = &cobra.Command{
 	Run:   LocalSmokeTest,
 }
 
+type localArgs struct {
+	contractsDeployed bool
+}
+
+var localTestArgs = localArgs{}
+
 func init() {
 	RootCmd.AddCommand(LocalCmd)
+	LocalCmd.Flags().BoolVar(&localTestArgs.contractsDeployed, "deployed", false, "set to to true if running smoketest again with existing state")
```

Comment:
We could maybe also have persistence by default and use `reset` flag to redeploy contracts

Reply:
I think it is a good idea to be able to optionally redeploy but I noticed a lot of the tests failed for reasons that weren't so obvious. I would need some time to investigate and fix those issues. Perhaps in another task we can add this since we will be building more test workflows.